### ISSUE TITLE
TypeError: mappings.filter is not a function

### ISSUE THREAD

**Issue by tyshgc** [pantharshit00/prisma-docs-generator#19]
Hi! 
I use prisma-docs-generator 0.0.6 and Prisma 2.11.0, has this error. 

```
npx prisma generate
Environment variables loaded from /Users/tyshgc/development/tyshgc/study_prisma/starter/prisma/.env
Prisma schema loaded from prisma/schema.prisma
Error: 
TypeError: mappings.filter is not a function
    at getMappings (/Users/tyshgc/development/tyshgc/study_prisma/starter/node_modules/prisma-docs-generator/dist/generator/transformDMMF.js:17:10)
    at Object.transformDMMF [as default] (/Users/tyshgc/development/tyshgc/study_prisma/starter/node_modules/prisma-docs-generator/dist/generator/transformDMMF.js:11:19)
    at Object.onGenerate (/Users/tyshgc/development/tyshgc/study_prisma/starter/node_modules/prisma-docs-generator/dist/index.js:38:45)
    at LineStream.<anonymous> (/Users/tyshgc/development/tyshgc/study_prisma/starter/node_modules/@prisma/generator-helper/dist/generatorHandler.js:13:46)
    at LineStream.emit (events.js:314:20)
    at addChunk (_stream_readable.js:303:12)
    at readableAddChunk (_stream_readable.js:279:9)
    at LineStream.Readable.push (_stream_readable.js:218:10)
    at LineStream.Transform.push (_stream_transform.js:166:32)
    at LineStream._pushBuffer (/Users/tyshgc/development/tyshgc/study_prisma/starter/node_modules/@prisma/generator-helper/dist/byline.js:108:23)
```

In `getMappings()` of [transformDMMF.ts](https://github.com/pantharshit00/prisma-docs-generator/blob/69d32c72faa6bd76796f15a6354f2069cbd3066b/src/generator/transformDMMF.ts#L37-L38), The argument `mappings` is an object, not an array?

**Comment by pantharshit00** [comment]
Looks like there was some breaking change in the dmmf format. I will fix this soon when I get some time

**Comment by pantharshit00** [comment]
Fixed via #21, doing a release now

**Comment by pantharshit00** [comment]
Upgrade to v0.1.0 https://www.npmjs.com/package/prisma-docs-generator/v/0.1.0 :)
